### PR TITLE
Implement python change case program

### PR DESCRIPTION
**Pull request by SachinPrasanth777**
### Description
implement python change case program
Screenshot 2023-10-20 at 8 17 16 PM

### CODE REVIEW

**Comment by shawshankkumar** [comment]
always try to have commit messages in lower case.

**Comment by shawshankkumar** [comment]
assign these and add labels:
image

**Comment by SachinPrasanth777** [comment]
I'm not a collaborator...so will I be able to add

**Comment by SachinPrasanth777** [comment]
I have changed the title of the pull request to lower case.

**Comment by SachinPrasanth777** [comment]
I have changed the commit message to lower case

File: .DS_Store
Comment (sm-sami) [inline .DS_Store:1]: Delete this file

**Review by sm-sami — CHANGES_REQUESTED** [review]

**Review by sm-sami — APPROVED** [review]
LGTM!

**Pull request merged**